src/components/DayPicker.jsx

```diff
@@ -842,7 +859,7 @@ class DayPicker extends React.PureComponent {
     });
   }
 
-  renderNavigation() {
+  renderNavigation(navDirection = null) {
```

Comment:
curious, why default it to null? this only has the effect of making `undefined` always be `null`, but it's not clear to me why that would matter.

Reply:
you're right, it doesn't matter. removing the default